core-web/libs/edit-content/src/lib/fields/dot-edit-content-json-field/dot-edit-content-json-field.component.html

```diff
@@ -0,0 +1,6 @@
+<div class="json-field__container" data-testId="json-field-container" style="resize: 'vertical'">
```

Comment:
Is this inline style necessary?

Reply:
We can do this in the `:host` for sure :)

Reply:
I added that inline style to test it

Reply:
Done!